src/main/java/org/opentripplanner/street/model/edge/StreetEdge.java

```diff
@@ -1106,6 +894,14 @@ protected void setElevationExtensionUsingParent(
     StreetElevationExtension.addToEdge(this, profile, true);
   }
 
+  short getFlags() {
+    return flags;
+  }
+
+  int getMillimeterLength() {
```

Comment:
We could use `Distance` here - so far it's only used by the fares but it can be generalized.

Reply:
While refactoring it made things a bit easier in some places to get the conversion meter/millimeter right. 
But no strong opinion about it, we can discuss this in the meeting.